packages/core/src/login/webview/vue/backend.ts

```diff
@@ -61,6 +61,22 @@ export abstract class CommonAuthWebview extends VueWebview {
         return globals.regionProvider.getRegions().reverse()
     }
 
+    private didCall: { login: boolean; reauth: boolean } = { login: false, reauth: false }
+    public setUiReady(state: 'login' | 'reauth') {
```

Comment:
is this something we could just use the once() util for? That way you don't have to keep track of the extra state?

Reply:
I wanted to emit once for `login` and `reauth` separately which didn't allow me to use `once()`. But I think in a future PR we could do some memoize + once to address this